gems/aws-sdk-core/3/aws.rbs

```diff
@@ -0,0 +1,4 @@
+module Aws
+  def self.config: () -> Hash[Symbol, untyped]
```

Comment:
More precisely, it seems to be `attr_reader`.
https://github.com/aws/aws-sdk-ruby/blob/62e0113f7b9efe66bb77318ae85b147ed462a55c/gems/aws-sdk-core/lib/aws-sdk-core.rb#L119
```suggestion
  attr_reader self.config: Hash[Symbol, untyped]
```